WikiWatershed/model-my-watershed
# Cleanup, consistent font sizes

## Overview

This commit provides general style cleanup for the new application layout. In addition, it attempts to standardize font sizes across the application.

Connects #XXX

### Demo

![image](https://cloud.githubusercontent.com/assets/1809908/26459800/b277683a-4145-11e7-9f12-88af9f41860f.png)

![image](https://cloud.githubusercontent.com/assets/1809908/26459810/be6e14ea-4145-11e7-88d9-ef540e984ce0.png)

![image](https://cloud.githubusercontent.com/assets/1809908/26459835/cde7ae18-4145-11e7-977e-5a7de16f9b5e.png)

## Testing Instructions

 * Try to look at every part of the application to see if any of the style is busted
